src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -46,7 +46,7 @@ public class AggresiveCacheOptions
 
         private Topology _topology;
         private readonly Timer _updateTopologyTimer;
-        private bool _firstTimeTryLoadFromTopologyCache = true;
+        private readonly bool _firstTimeTryLoadFromTopologyCache = true;
```

Comment:
Why is this even needed now? We are only using it once, and we already know how it behaves.